cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/service/http/HttpHandlerGeneratorTest.java

```diff
@@ -261,33 +309,119 @@ protected FileUploadHandler createHandler() {
       HttpURLConnection urlConn = (HttpURLConnection) new URL(
         String.format("http://%s:%d/content/upload/test.txt",
                       bindAddress.getHostName(), bindAddress.getPort())).openConnection();
+      try {
+        urlConn.setReadTimeout(2000);
+        urlConn.setDoOutput(true);
+        urlConn.setRequestMethod("PUT");
+        // Set to use default chunk size
+        urlConn.setChunkedStreamingMode(-1);
+
+        // Write over 150MB of data
```

Comment:
This is due to the default 150MB memory buffer used by netty-http server. We wrote > 150M of data to make sure we go over that 150MB limit without failure.